_posts/en/newsletters/2021-12-01-newsletter.md

```diff
@@ -0,0 +1,99 @@
+  upgrade to a bug fix release, 0.14.1 (described in the *release*
+  section below).
+
+    A related [discussion][xraid interop] was started on the
+    Lightning-Dev mailing list about improving interoperability testing.
+    Christian Decker, who previously created and maintained the LN
+    [integration testing framework][ln integration], said that [he
+    believes][decker interop] basic interoperability testing is
+    "unlikely to catch any but the most egregious issues".  LN
+    developers participating in the discussion [noted][zmn interop] that
+    catching these sort of bugs is why every major implementation
+    provides release candidates (RCs) and why expert users and
+    administrators of production systems are encouraged to contribute to
+    the testing of those RCs.
+
+    For anyone interested in participating in such testing, the Optech
+    newsletter lists RCs from four major LN implementations, plus
+    various other Bitcoin software.  The LND version under discussion was
+    listed as an RC in Newsletters [#174][news174 lnd] and [#175][news175
+    lnd].
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.14.1-beta][] is a maintenance release that fixes the issue
+  described in more detail both in the *news* section above and in the
+  description of [LND #6026][] in the *notable changes* section below.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16807][] Updates address validation to return the indices of
+  likely invalid characters in [Bech32 and Bech32m][topic bech32] addresses.
+  The errors will be correctly identified if no more than two substitution
+  errors were made. The pull request also improves test coverage, adds more
+  documentation to the address validation code, and improves error messages
+  when decoding fails, especially to distinguish use of Bech32 and Bech32m.
+
+- [Bitcoin Core #22364][] adds support for creating taproot descriptors. This
+  allows wallet users to generate and use P2TR addresses by creating a default
+  bech32m descriptor with their wallet instead of importing one.
+
+- [LND #6026][] fixes an issue with the [implementation][lnd #5669] of
+  [BOLTs #880][] explicit channel type negotiation (see [Newsletter
+  #165][news165 bolts880]). A [proposed change][bolts #906] to the LN
+  specification will allow LND to eventually implement strict explicit
```

Comment:
Given that it's still being proposed, "will" sounds a little to inevitable, especially given that many other Lightning developers seem to have strong diverging opinions on this matter.

```suggestion
  specification would allow LND to eventually implement strict explicit
```